### PR TITLE
finished exp milestone-3

### PR DESCRIPTION
**Pull request by JayKowski**
- All the recommended styles have been applied and milestone is ready for review.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
+
+  const wideStyle = {
+    width: '49.7%',
+    backgroundColor: color,
+  };
+
+  const normalStyle = {
+    width: '24.7%',
+    backgroundColor: color,
+  };
```
Comment (bellom) [inline src/components/Button.js]: We should use `25%` for all buttons and `50%` for the `0` button in here.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896093/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @JayKowski  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**1. Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file and make sure to add your live link from Heroku as stated in milestone one.**
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**_Seems you forgot you made a promise to build up your README file 😄_** 
image


**2. Kindly check out my comments below;**


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/components/App.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.App {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+}
```
Comment (shloch) [inline src/components/App.css:5]: You can center the app, for a symmetry with the page 
``` 
  margin: auto 
```

**Review by shloch — APPROVED** [review]
## Hello @JayKowski , 

Thanks for bringing the corrections
Screen Shot 2020-09-24 at 5 29 18 PM

App still works and is unbroken,
keep up with the good work
Happy coding

File: src/components/App.css (hunk 2 above)
Comment (JayKowski) [reply src/components/App.css:5]: Hey Shey,

Thanks for the feedback. I will center the project on the next milestone submission👍

**Pull request merged**